specification/hybridconnectivity/PublicCloud.Management/tspconfig.yaml

```diff
@@ -38,6 +38,12 @@ options:
     generate-fakes: true
     head-as-boolean: true
     inject-spans: true
+  "@azure-tools/typespec-java":
+    service-dir: "sdk/hybridconnectivity"
+    package-dir: "azure-resourcemanager-hybridconnectivity"
+    namespace: "com.azure.resourcemanager.hybridconnectivity"
```

Comment:
put the namespace to client.tsp

Reply:
Fixed it by [195f4c8](https://github.com/Azure/azure-rest-api-specs/pull/32115/commits/195f4c823a4f487551916a49b3c83d64d4d74762)